src/v2/BaseLoginRouter.js

```diff
@@ -80,9 +79,8 @@ export default Router.extend({
     this.appState = new AppState();
 
     const wrapper = new AuthContainer({ appState: this.appState });
-    const skipLink = new SkipLink();
 
-    $(options.el).append([skipLink.render().$el, wrapper.render().$el]);
+    $(options.el).append(wrapper.render().$el);
```

Comment:
Can you add the screenshot for V2 login-page? I don't think it's required in V2 yet, we don't have consent for v2 yet.. i guess. @haishengwu-okta  ??

Reply:
ah.. is't just the previous code removal ?

Reply:
Yes just removing the code I added.

Reply:
Correct, no consent in v2 yet.